### DISCUSSION TITLE
Cant access audiobookshelf interface "This site can’t be reached synology-ip-address’s DNS address could not be found

### DISCUSSION THREAD

**Discussion by vicdagod** [advplyr/audiobookshelf#1885] [Q&A] [answered]
Hello all. I spent about 30 minutes looking for similar problems/discussions, but I dont see any. This is my first time using docker. 
I tried 2 methods to installing audiobookshelf. I tried both multiple times

1. https://mariushosting.com/how-to-install-audiobookshelf-on-your-synology-nas/
2. https://neellik.com/how-to-install-audiobookshelf-on-your-synology-nas-with-docker/

both methods were quite similar.

after the docker appears to be created and I verified it is running, I attempt to access the interface via 
http://synology-ip-address:13378/ or http://synologyip:13378

and Im met with 
"
This site can’t be reached
synology ip’s DNS address could not be found. Diagnosing the problem.
DNS_PROBE_POSSIBLE
"

email via task scheduler:
"
Task: Install Audiobookshelf
Start time: Fri, 30 Jun 2023 19:33:35 -0400
Stop time: Fri, 30 Jun 2023 19:33:37 -0400
Current status: 0 (Normal)
Standard output/error:
a6d0e1eb067acbfe6a0ce5d1b78fa7a96720819043a629e9709e7bc0f160febb
"

Im not sure what the issue is at all. I verified the current user has read/write permissions of the docker shared folder.
Im 100% sure I linked to the correct folders that I created.

my user defined script is:

docker run -d --name=audiobookshelf \
-p 13378:80 \
-e PUID=1026 \
-e PGID=100 \
-v /volume1/docker/audiobookshelf/config:/config \
-v /volume1/music/audiobooks:/audiobooks \
-v /volume1/docker/audiobookshelf/metadata:/metadata \
--restart always \
advplyr/audiobookshelf

I verified ID and group were correct via cmd, SSH [putty] and also verrified using a scheduled task
Any help or even another method would be appreciated. Thanks

**Comment by advplyr** [comment, marked as answer]
Are you putting your actual IP address in place of "synology-ip-address"?

It won't break anything but `PUID` and `PGID` are unnecessary, they won't do anything.
